package.json

```diff
@@ -14,6 +15,7 @@
     "url": "https://github.com/sendbird/chat-ai-widget.git"
   },
   "scripts": {
+    "install:deps": "npm install --force && git submodule update --init",
```

Comment:
Why we're using `--force` param here? There's a peer dep setting in `@mojojoejo/vite-plugin-purgecss` like below, so we can't use the latest vite version. So I put `--force` option to ignore the error.
![Screenshot 2024-04-29 at 5 56 51 PM](https://github.com/sendbird/chat-ai-widget/assets/10060731/f15eb1fb-8b36-47fc-ac66-7b03e1fb1738)

Reply:
Could you test if it also works using the Rollup plugin instead of the Vite plugin?
- https://v1.purgecss.com/with-rollup

Reply:
@bang9  Hmm it seems it also conflicts with other packages and doesn't give a happy result. Let me split this purgecss task and do it separately :) 

I'll put the purgecss related stuff.

Reply:
😭

Reply:
If the reduction in bundle size for each Vite version is not significantly noticeable, it might be better to maintaining version compatibility